src/components/DateInput.jsx

```diff
@@ -79,6 +79,10 @@ const defaultProps = {
 
   // accessibility
   isFocused: false,
+  ariaLabel: undefined,
+  ariaInvalid: undefined,
```

Comment:
shouldn't this be false, since it's a boolean?
```suggestion
  ariaInvalid: false,
```

Reply:
If we set this to `false` then the attribute will always appear in the DOM. Is that the behavior we want? If not set, the default is `false`.

I'm now also realizing this property can accept a couple string values - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

Reply:
ah, true. in that case `undefined` is the proper default, and i'm glad to see the more accurate propType :-)